### PR TITLE
Additions to Gumbel Rounding

### PR DESCRIPTION
**Pull request by ofirgo**
The following additions to Gumbel Rounding, GPTQ, and Jacobian weights computation:
* Normalization to weights using log scale.
* Added option for different gumbel scale values per bit-width.
* Moved gumbel_scale parameter to GumbelConfig instead of GradientPTQConvifg.
* Added tqdm to model_gradients random interest points and GPTQ training loops.

### CODE REVIEW

**Review by eladc-git — APPROVED** [review]

File: model_compression_toolkit/qunatizers_infrastructure/keras/quantize_wrapper.py [hunk 1]
```diff
@@ -21,6 +21,9 @@
 
 if FOUND_TF:
     import tensorflow as tf
+
+    keras = tf.keras
```
[the remark this answers is not in the document]
Comment (ofirgo) [reply model_compression_toolkit/qunatizers_infrastructure/keras/quantize_wrapper.py:25]: My bad, it was a fix for a problem that I got when merging from main, but it was already taken care of in a separate PR, I'll remove this.
